managed/services/server/updater_test.go

```diff
@@ -252,7 +252,7 @@ func TestUpdater(t *testing.T) {
 		latest, err := u.latest(context.Background())
 		require.NoError(t, err)
 		assert.NotNil(t, latest)
-		assert.True(t, strings.HasPrefix(latest.Version.String(), "2.41."), "latest version of PMM 2 should have prefix 2.41.")
+		assert.True(t, strings.HasPrefix(latest.Version.String(), "2."), "latest version of PMM 2 should have prefix 2.")
```

Comment:
LGTM

Shouldn't we add the same `|| strings.HasPrefix(res.Payload.Installed.Version, "3.")` to be prepared for v3?